redfish-core/lib/log_services.hpp

```diff
@@ -528,6 +548,19 @@ inline void
                                 size = *sizePtr;
                                 break;
                             }
+                            else if (propertyMap.first == "InitiatedBy")
+                            {
+                                const std::string* session =
+                                    std::get_if<std::string>(
+                                        &propertyMap.second);
+                                if (session == nullptr)
+                                {
+                                    messages::internalError(asyncResp->res);
+                                    break;
+                                }
+                                sessionId = *session;
+                                break;
```

Comment:
Why the break ?

Reply:
yeah, will take this in a new commit may be..